source/cipher_openssl.c

```diff
@@ -533,6 +534,7 @@ int aws_cryptosdk_sig_sign_start(
     AWS_PRECONDITION(AWS_OBJECT_PTR_IS_WRITABLE(ctx));
     AWS_PRECONDITION(AWS_OBJECT_PTR_IS_READABLE(alloc));
     AWS_PRECONDITION(AWS_OBJECT_PTR_IS_READABLE(props));
+    AWS_PRECONDITION(priv_key != NULL);
```

Comment:
Why can't we assume here `aws_string_is_valid()`?

Reply:
Updated!